truss/templates/custom/model/model.py

```diff
@@ -1,19 +1,30 @@
-from typing import Any
+"""
+This file is where the key logic for serving your model is defined.
```

Comment:
This is a good help message, I'm just going to update it quick before merge.

Reply:
cool -- go for it!

Reply:
```
The `Model` class is an interface between the ML model that you're packaging and the model server that you're running it on.

The main methods to implement here are:
* `load`: runs exactly once when the model server is spun up or patched and loads the model onto the model server. Include any logic for initializing your model, such as downloading model weights and loading the model into memory.
* `predict`: runs every time the model server is called. Include any logic for model inference and return the model output.

See https://truss.baseten.co/quickstart for more.
```

Reply:
```
The `Model` class is an interface between the ML model that you're packaging and the model server that you're running it on.

The main methods to implement here are:
* `load`: runs exactly once when the model server is spun up or patched and loads the model onto the model server. Include any logic for initializing your model, such as downloading model weights and loading the model into memory.
* `predict`: runs every time the model server is called. Include any logic for model inference and return the model output.

See https://truss.baseten.co/quickstart for more.
```